src/pages/Missions.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from 'react';
-import { Table } from 'react-bootstrap';
+import { Badge, Button, Table } from 'react-bootstrap';
 import { useDispatch, useSelector } from 'react-redux';
-import { getMissions } from '../redux/missions/missionsSlice';
+import { getMissions, joinMission } from '../redux/missions/missionsSlice';
 
 const Missions = () => {
   const { missions } = useSelector((state) => state.missions);
```

Comment:
Optional: You could consider destructuring the missions object. This gives a much cleaner code.

Reply:
Thank you for the suggestion. I'm going to consider that option.